.github/pull_request_template.md

```diff
@@ -7,6 +7,6 @@ tracking issue or there are none, feel free to ignore this.
 This PR will get automatically assigned to a reviewer. In case you would like
 a specific user to review your work, you can assign it to them by using
 
-    r​? <reviewer name>
```

Comment:
This is actually

```
    r⌴? <reviewer name>
```

Can check with tools like <https://www.dcode.fr/zero-width-space>.